search/src/test/scala/weco/api/search/fixtures/ApiFixture.scala

```diff
@@ -83,20 +84,51 @@ trait ApiFixture extends AnyFunSpec with ScalatestRouteTest with IndexFixtures {
     routes: Route,
     path: String,
     unordered: Boolean = false
-  )(expectedResponse: (StatusCode, String)) =
-    eventually {
-      expectedResponse match {
-        case (expectedStatus, expectedJson) =>
+  )(expectedResponse: (StatusCode, String)): Assertion =
+    expectedResponse match {
+      case (expectedStatus, expectedJson) =>
+        def responseJson: Json = eventually {
           Get(path) ~> routes ~> check {
             contentType shouldEqual ContentTypes.`application/json`
-            parseJson(responseAs[String], unordered) shouldEqual parseJson(
-              expectedJson,
-              unordered
-            )
             status shouldEqual expectedStatus
+            parseJson(responseAs[String], unordered)
           }
+        }
+        responseJson shouldEqual parseJson(expectedJson, unordered)
```

Comment:
I have moved the "check the json" outside of the eventually.

What was happening before was that a failing test would take ages to execute, because it would exhaust all of its `eventually` retries checking whether the returned JSON matched.  This also gave a cryptic failure message about timing out, when the real problem is that the data is wrong.

What we really want is "eventually we get a response" then "the response is...".  That way, a request that successfully returns the wrong data can fail promptly and with the right reason.